icu_benchmarks/models/custom_metrics.py

```
@@ -143,3 +148,90 @@ def confusion_matrix(y_true: ndarray, y_pred: ndarray, normalize=False) -> torch
         for j in range(confusion.shape[1]):
             confusion_dict[f"class_{i}_pred_{j}"] = confusion[i][j]
     return confusion_dict
+
+
+def matthews_correlation_coefficient(y_true: ndarray, y_pred: ndarray, normalize=False) -> float:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)